### DISCUSSION TITLE
How and when the database file and the tables is created needs to be documented

### DISCUSSION THREAD

**Discussion by danielo515** [withastro/docs#8978] [Ideas]
Hello.
I wanted to use astro-db for a very simple project and it looked very convenient. However I am not invited to the astro studio, so I decided to deploy a docker container.
After some struggle I managed to get the thing working with an attacher volume, and the server starts, but then I get a runtime error about the tables not existing.
When are databases created? I assumed they were created on server start, but that is not obviously the case, so when? How can I ensure the tables exist on my first server start?

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

Sorry no one could answer your question before but, since [Astro DB has been removed in Astro 7](https://docs.astro.build/en/guides/upgrade-to/v7/#removed-astrojsdb), I'm going to close this discussion.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
